src/pages/getting_started.tsx

```diff
@@ -1,46 +1,55 @@
-import { UserLayout } from '@layout';
-import { NextPage } from 'next';
-import { Card } from 'react-bootstrap';
+import { UserLayout } from "@layout";
+import { NextPage } from "next";
+import { Card } from "react-bootstrap";
 
 const GettingStarted: NextPage = () => (
   <UserLayout>
+    <Card.Title>Getting Started</Card.Title>
+    <Card.Text>
+      Follow the instructions below to install the Martian environment, upload
+      your algorithm submissions, request scrimmages with other players, and
+      check out your match results - happy hacking!
+    </Card.Text>
     <Card className="mb-3">
       <Card.Body>
-        <Card.Title>Getting Started</Card.Title>
+        <Card.Title>Installation</Card.Title>
         <Card.Text>
-          Follow through the instructions below to learn more about installation
-          instructions, how you can upload your bot submissions, request
-          scrimmages with other players, and check out your match results!
+          Visit our AWAP 2023{" "}
+          <a href="https://github.com/ACM-CMU/awap-viewer-2023">Viewer</a> and{" "}
```

Comment:
they don't have to install viewer. they just need to go to view.awap.acmatcmu.com

Reply:
fixed - will they need to install engine?

Reply:
they will need to install python and the relevant packages to run the engine